tests/load/locust_tests/locustfile.py

```
@@ -42,25 +42,28 @@
 # affecting the client's Suggest experience
 CLIENT_VARIANTS: str = ""
 
-# See RemoteSettingsGlobalSettings in
-# https://github.com/mozilla-services/merino/blob/main/merino-settings/src/lib.rs
-KINTO__SERVER_URL = os.environ["KINTO__SERVER_URL"]
-
-# See default values in RemoteSettingsConfig in
-# https://github.com/mozilla-services/merino/blob/main/merino-settings/src/providers.rs
-KINTO__BUCKET = os.environ["KINTO__BUCKET"]
-KINTO__COLLECTION = os.environ["KINTO__COLLECTION"]
+# See Ops - Remote Settings
+# https://github.com/mozilla-services/merino-py/blob/main/docs/ops.md#remote_settings
+MERINO_REMOTE_SETTINGS__SERVER: str | None = os.getenv(
+    "MERINO_REMOTE_SETTINGS__SERVER", os.getenv("KINTO__SERVER_URL")
```

I like the fallbacks to the old environment variables! Will `get_adm_queries` throw, or otherwise do the right thing, if any of them are `None`—like, if you forget to specify one of them?

---

Yes, it the backend modules we re-use will raise ValueErrors and the locust tests will log and fail fast from a catch.